src/general/OptionsMenu.cs

```diff
@@ -269,6 +269,9 @@ public class OptionsMenu : ControlWithInput
     [Export]
     public NodePath ScreenshotDirectoryWarningBoxPath = null!;
 
+    [Export]
+    public NodePath ScreenEffectButtonPath = null!;
```

Comment:
Variables should be grouped based on the tab (and after where this is visually).